From f7ac48e020e43df2c885c45b1b73caf1e4b56b90 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Tue, 18 Feb 2014 19:08:18 +0200
Subject: [PATCH] dont check for libbluetooth. HAVE_LIBBLUETOOTH undefined is
 fine

or bluetooth kernel modules will be loaded as soon as xbmc starts,
even if there is no bt hardware present.
---
 configure.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.in b/configure.in
index 42dd00e..de12212 100644
--- a/configure.in
+++ b/configure.in
@@ -1144,7 +1144,6 @@ AC_CHECK_LIB([ssl],         [main],, AC_MSG_ERROR($missing_library))
 if test "$use_mysql" = "yes"; then
   AC_CHECK_LIB([mysqlclient], [main],, AC_MSG_ERROR($missing_library))
 fi
-AC_CHECK_LIB([bluetooth],   [hci_devid],, AC_MSG_RESULT([Could not find suitable version of libbluetooth]))
 AC_CHECK_LIB([yajl],        [main],, AC_MSG_ERROR($missing_library))
 AC_CHECK_LIB([tinyxml],     [main],, AC_MSG_ERROR($missing_library))
 if test "$target_platform" = "target_android" ; then
-- 
1.9.rc1

